scripts/README.md

```diff
@@ -6,11 +6,14 @@ Use at your own risk!
 
 ### Prerequisites
 
-Contracts must be compiled and, if working on any non-local network, deployment addresses must be injected into built contracts.
+The scripts require the following software installed: [git](https://git-scm.com/), [yarn](https://yarnpkg.com/) and [node with version 10](https://nodejs.org/en/blog/release/v10.18.0/).
+
+The Gnosis Protocol contracts must be compiled and the deployment addresses must be injected into built contract-artifacts.
 To do so, run:
 
 ```
-npx truffle compile
+yarn global add npx
```

Comment:
Npx comes with a Node installation since version 5 ([source](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner)), so this line is not needed.